### DISCUSSION TITLE
关于近期CurseForge上部分模组被恶意篡改的问题

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#1989] [提问] [answered]
根据可靠的消息，部分于curseforge的mod被人篡改并且携带了后门
https://discord.com/channels/774352792659820594/915326508342575145/1115815452074061884
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/da1beba6-001a-4b58-8d7f-089c42537a87)

https://www.bilibili.com/read/cv24184930
所以说PCL这边有没有必要说明一下，例如发个公告提醒一下用户🤔

**Comment by MoYuan-CN** [comment, marked as answer]
见 #1990
